app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -422,6 +427,17 @@ def parse_node_condition(condition)
       }
     end
 
+    def parse_pod_condition(condition)
+      {
+        :name                 => condition.type,
+        :status               => condition.status,
+        :last_heartbeat_time  => condition.lastHeartbeatTime,
+        :last_transition_time => condition.lastTransitionTime,
+        :reason               => condition.reason,
+        :message              => condition.message
+      }
+    end
```

Comment:
> This one is identical to parse_node_condition (because conditions are the same entity). Please unify this with parse_node_condition inside parse_conditions as suggested above.

Done.